go/vt/vttest/local_cluster.go

```diff
@@ -237,6 +245,16 @@ func (db *LocalCluster) Setup() error {
 
 	log.Infof("LocalCluster environment: %+v", db.Env)
 
+	// Set up topo manager if we are using a remote topo server
+	if db.RemoteTopoImplementation != "" {
+		db.topo = db.Env.TopoManager(db.RemoteTopoImplementation, db.RemoteTopoGlobalServerAddress, db.RemoteTopoGlobalRoot, db.Topology)
+		log.Infof("Initializing Topo Manager (%T)...", db.topo)
```

Comment:
This will print the golang type. Is that what we want here? I would think this will always be `TopoManager` but we are interested in the implementation details?

Reply:
good catch. fixed to use `%+v`